apps/src/code-studio/components/AdvancedShareOptions.jsx

```diff
@@ -129,6 +135,7 @@ class AdvancedShareOptions extends React.Component {
     return (
       <div>
         <p style={style.p}>{i18n.shareEmbedDescription()}</p>
+        <p style={style.warningp}>{i18n.shareEmbedWarning()}</p>
```

Comment:
Nit: rather than copy over all of the existing `p` styles in your new styles object, you could simplify your new styles to just include the color (ie, `warning: {color: color.red}}`, then do something like this when you use it:

```
<p style={{
  ...style.p,
  ...style.warning
}}>{i18n.shareEmbedWarning()}</p>
```

This is "spread syntax", which can be a nice way to combine objects. More [info here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#spread_in_object_literals)